src/client/event_handler.rs

```diff
@@ -139,6 +140,10 @@ event_handler! {
     /// The old channel data is only provided when the cache feature is enabled.
     async fn channel_update(&self, ChannelUpdate { ctx: Context, old: Option<Channel>, new: Channel });
 
+    /// Disptahced when a new audit log entry is created
+    ///
```

Comment:
Typo that was missed before merge. Also best to put a period at the end. The extra line is also unnecessary.

Reply:
[Manually fixed](https://github.com/serenity-rs/serenity/commit/b792c981d965644a6920ae3d47d9889c1da99520).